docs/01-Using-Fleet/standard-query-library/standard-query-library.yml

```
@@ -918,4 +918,89 @@ spec:
+      description: "Checks for the existence of a GPO that requires the Windows Firewall - Private Profile is enabled."
+      resolution: "Contact your IT administrator to ensure your Windows system is receiving a GPO that requires that the Windows Firewall - Private profile is enabled."
+      platforms: Windows
+      tags: compliance, CIS, CIS9.2.1
+      platform: windows
+      contributors: defensivedepth
+    ---
+    apiVersion: v1
+    kind: policy
+    spec:
+      name: Ensure Windows Firewall - Public Profile is enabled (Windows)
+      query: SELECT 1 FROM registry WHERE path LIKE 'HKEY_LOCAL_MACHINE\Software\Policies\Microsoft\WindowsFirewall\PublicProfile\EnableFirewall' AND CAST(data as integer) = 1;
+      description: "Checks for the existence of a GPO that requires the Windows Firewall - Public Profile is enabled."
+      resolution: "Contact your IT administrator to ensure your Windows system is receiving a GPO that requires that the Windows Firewall - Public profile is enabled."
+      platforms: Windows
+      tags: compliance, CIS, CIS9.3.1
+      platform: windows
+      contributors: defensivedepth
+    ---
+    apiVersion: v1
+    kind: policy
+    spec:
+      name: Ensure SMBv1 client driver is disabled (Windows)
+      query: SELECT 1 FROM windows_optional_features WHERE name = 'SMB1Protocol-Client' AND state != 1;
+      description: "Checks that the SMBv1 client is disabled."
+      resolution: "Contact your IT administrator to discuss disabling SMBv1 on your system."
+      platforms: Windows
+      tags: compliance, CIS, CIS18.3.2
+      platform: windows
+      contributors: defensivedepth
+    ---
+    apiVersion: v1
+    kind: policy
+    spec:
+      name: Ensure SMBv1 server is disabled (Windows)
+      query: SELECT 1 FROM windows_optional_features WHERE name = 'SMB1Protocol-Server' AND state != 1
+      description: "Checks that the SMBv1 server is disabled."
+      resolution: "Contact your IT administrator to discuss disabling SMBv1 on your system."
+      platforms: Windows
+      tags: compliance, CIS, CIS18.3.3
+      platform: windows
+      contributors: defensivedepth
+    ---
+    apiVersion: v1
+    kind: policy
+    spec:
+      name: Ensure LLMNR is disabled (Windows)
+      query:  SELECT 1 FROM registry WHERE path LIKE 'HKEY_LOCAL_MACHINE\SOFTWARE\Policies\Microsoft\Windows NT\DNSClient\EnableMulticast' AND CAST(data as integer) = 0;
+      description: "Checks for the existence of a GPO that disables LLMNR."
+      resolution: "Contact your IT administrator to discuss disabling LLMNR on your system."
+      platforms: Windows
+      tags: compliance, CIS, CIS18.5.4.2
+      platform: windows
+      contributors: defensivedepth
+    ---
+    apiVersion: v1
+    kind: policy
+    spec:
+      name: Ensure OS Automatic Updates is enabled (Windows)
```

```suggestion
      name: Automatic updates enabled (Windows)
```

Same "naming convention" request.